### PR TITLE
XTCE Generation Fixes

### PR DESCRIPTION
**Pull request by dmitri-mcguckin**
See Issue #9 for the complete list of issues that are addressed in this PR.

### CODE REVIEW

File: oresat_configs/scripts/gen_xtce.py [hunk 1]
```diff
@@ -271,7 +290,7 @@ def write_xtce(config: OreSatConfig, dir_path: str = "."):
     # write
     tree = ET.ElementTree(root)
     ET.indent(tree, space="  ", level=0)
-    file_name = f"{config.oresat_id.name.lower()}-xtce.xml"
+    file_name = f"{config.oresat_id.name.lower().replace('_', '.')}.xtce"
```
[the remark this answers is not in the document]
Comment (ryanpdx) [reply oresat_configs/scripts/gen_xtce.py:293]: Do we want this as `.xtce`? leaving it as `-xtce,xml` helps when opening it with IDEs and/or text editors

[the remark this answers is not in the document]
Comment (dmitri-mcguckin) [reply oresat_configs/scripts/gen_xtce.py:293]: FWIW xtce is recognized as an xml file-ending for my editor and it's a bit more terse for what it should be.

I'd prefer the file name format this way.

[the remark this answers is not in the document]
Comment (dmitri-mcguckin) [reply oresat_configs/scripts/gen_xtce.py:293]: That or most editors these days scan the content to detect file-type anyways.

I know both neovim and vscode do this- anyhow.

[the remark this answers is not in the document]
Comment (ryanpdx) [reply oresat_configs/scripts/gen_xtce.py:293]: Okay

The `.` in file names for non file extensions still bothers me, but whatever

**Review by ryanpdx — APPROVED** [review]

File: oresat_configs/scripts/gen_xtce.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (dmitri-mcguckin) [reply oresat_configs/scripts/gen_xtce.py:293]: I'm fine with leaving the underscores, as long as it ends in `*.xtce`

**Pull request merged**